README.md

```diff
@@ -1,46 +1,98 @@
-HTML + CSS, mark-up and mark-down should all be formatted using prettier.
+As a standard, all code contributions should be linted with [ESLint](https://eslint.org/) using `.eslintrc.js` and formatted with [Prettier](https://prettier.io/). **Note:** HTML + CSS, mark-up and mark-down code are exemptions and should be formatted using [Prettier](https://prettier.io/) but do not need to be linted.
 
 `npm run lint` will lint the codebase.
 
 Our GitHub Actions will also lint any pull requests before they're merged.
 
 ## Testing
 
-Code should be tested using [Jest](https://jestjs.io/) and [React Testing Library](https://testing-library.com/docs/react-testing-library/intro/).
+[{{back to navigation}}](#table-of-contents)
+
+All code contributions should be tested using both the [Jest](https://jestjs.io/) and [React Testing Library](https://testing-library.com/docs/react-testing-library/intro/).
 
 `npm run test` will run any existing tests in our codebase.
 
-Our GitHub Actions will also test any pull requests before they're merged.
+Our GitHub Actions will also test any pull requests before they're merged! These all must pass and have 2 reviewers approval before a pull request can merge. If one or a few fail and your troubleshooting is not giving an answer, please check out the [gliff.ai Contribution Guide](https://github.com/gliff-ai/.github/blob/main/CONTRIBUTING.md) 👋 for further guidance.
+
+## Continuous Integration
+
+[{{back to navigation}}](#table-of-contents)
+
+GitHub Actions should only be included under the `.github/workflows` path.
+
+## Contribute
+
+[{{back to navigation}}](#table-of-contents)
 
-## Continuous integration
+We welcome all contributors and any contributions on this project through the likes of feedback on or suggesting features and enhancements, raising bug problems, reporting on security vulnerabilities, reviewing code, requesting or creating tests, user testing etc. to ensure gliff.ai can help enable the best and biggest positive impact possible. 
 
-GitHub Actions should be included under `.github/workflows`.
+Sounds good and want to contribute to the project? 🧑‍💻 \
+Please check the [gliff.ai Contribution Guide]((https://github.com/gliff-ai/.github/blob/main/CONTRIBUTING.md)) 👋 before you get started. Don’t forget the [gliff.ai Code of Conduct]((https://github.com/gliff-ai/.github/blob/main/CODE_OF_CONDUCT.md)) ⚠️ and  [gliff.ai Security Policy]((https://github.com/gliff-ai/.github/blob/main/SECURITY.md)) 🔒 too!
+
+A big thank you from the entire gliff.ai team to these fellow contributors ([emoji key](https://allcontributors.org/docs/en/emoji-key)): \
+[{{Contributor List - _coming soon_}}](https://github.com/all-contributors/all-contributors)
+
+## Contact
+
+[{{back to navigation}}](#table-of-contents)
+
+Need some help? 🤔 Have a question? 🧠 \
+Reach out to the gliff.ai team at [community@gliff.ai](mailto:community@gliff.ai?subject=[GitHub]) or on GitHub.
```

Comment:
What about issues? And do we want to use Github Discussion boards?

Reply:
@ChasNelson1990 I think having a Github Discussion board for each public repo is a great idea! Can you switch it on for this repo as an initial test?

Reply:
Done